### PR TITLE
google analytics only if not settings.DEBUG fixes #960

### LINKED ISSUE #960: Let's not Google Analytics locally
My Nightly blocks it locally anyway I noticed. 
I think, since we don't have a distinction between environments, let's simply make this depend on `settings.DEBUG`.

### PR DESCRIPTION
**Pull request by peterbe**
Fixes #960

### CODE REVIEW

File: app/experimenter/settings.py [hunk 1]
```diff
@@ -91,6 +91,8 @@
 
 WSGI_APPLICATION = "experimenter.wsgi.application"
 
+# https://docs.djangoproject.com/en/2.1/ref/settings/#internal-ips
+INTERNAL_IPS = config("INTERNAL_IPS", cast=Csv(), default="127.0.0.1")
```
Comment (mythmon) [inline app/experimenter/settings.py]: Is this related to Google Analytics?

Comment (peterbe) [reply app/experimenter/settings.py]: It's needed for the `django.template.context_processors.debug` context processor.

**Comment by peterbe** [comment]
I rewrote it so it's a dedicated setting for it. `True` by default to favor Production. 

It need a new context processor and none of the existing apps felt right so I started a new app. By the way, this new app `base`, is going to come in handy when we set up some system checks for example.

File: app/experimenter/base/tests/test_views.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
+from django.test import TestCase
+from django.urls import reverse
+
+
+class TestContextProcessors(TestCase):
+
+    def test_render_google_analytics_or_not(self):
+        headers = {settings.OPENIDC_EMAIL_HEADER: "user@example.com"}
```
Comment (mythmon) [inline app/experimenter/base/tests/test_views.py]: Even though it is the default, I'd prefer this block be in a `with self.settings(...)` block so that the settings this test rely on are explicit.

Comment (peterbe) [reply app/experimenter/base/tests/test_views.py]: That's an interesting topic. There is a `settings_test.py` which hardcodes a bunch of "defaults" specifically for the tests. 

So, when you're hacking on a test function like this you either have to memorize (or look up frequently) the contents of `settings_test.py`. *Or*, to save the code reader we *could* do things like set-what's-already-set here in the test to make it ultra explicit. 

I doubt there's a perfect solution. In lots of Django projects there are a bunch of assumptions that basically requires that you study the fixtures and/or the `settings.Test` class. 

E.g. https://github.com/mozilla-services/tecken/blob/9ea366093ee56f5e2607d1fe73ef90bb5d1ba8d2/tests/test_upload.py#L139-L150
This test could instead generate a zip on the fly there in the test instead of relying on knowing details.

**Comment by peterbe** [comment]
@mythmon squashed.
